cowprotocol/cowswap
# fix: do not trigger expired NPS for hidden orders

# Summary

Do not trigger expired NPS for hidden orders

# To Test

1. Clear your NPS cookie if already set
2. Set a short expiration time just so you don't have to wait a long time
3. Using a SC wallet, go all the way to request the order in the wallet, but do not sign/execute it
4. Wait for the order to expire
* In the device where the order was triggered, there should not be an expiration sound nor popup
* In the device where the order was triggered, NPS should not be displayed
* In another device with the same account, order will be displayed, and NPS might trigger as usual
